vmm/src/api/http_endpoint.rs

```diff
@@ -187,7 +187,9 @@ impl EndpointHandler for VmActionHandler {
             }
         } else {
             match self.action {
-                Boot => vm_boot(api_notifier, api_sender).map_err(HttpError::VmBoot),
+                Boot(stop_vm) => {
```

Comment:
Rather than extend the API at this point (and expose). Would you consider always stopping the VM if `--gdb` is specified instead?

Reply:
I removed `stop_vm` and added `stop_on_boot` to `Vm` in [7104f80](https://github.com/cloud-hypervisor/cloud-hypervisor/pull/3575/commits/7104f80cb5020a7ec54f46045277bb27bf08ea10).